test/fixtures/games_fixture.csv

```diff
@@ -0,0 +1,12 @@
+game_id,season,type,date_time,away_team_id,home_team_id,away_goals,home_goals,venue,venue_link
```

Comment:
We wanted to pull out a selection that had both postseason and regular season games for the fixtures.